CMakeLists.txt

```diff
@@ -198,3 +217,58 @@ add_custom_target(clean-generated
     COMMAND ${CMAKE_COMMAND} -E remove_directory ${GENERATED_DIR}
     COMMENT "Cleaning generated parser files"
 )
+
+# ------------------------------------------------------------
```

Comment:
This adds the option of building the shared library with the Julia glue code. By default this is off.